h/static/scripts/test/root-thread-test.js

```diff
@@ -179,6 +181,53 @@ describe('rootThread', function () {
     ]);
   });
 
+  describe('when the thread filter query is set', function () {
+    it('generates a thread filter function to filter out annotations', function () {
+      fakeBuildThread.reset();
+      fakeBuildThread.annotation = {target: [{ selector: {} }]};
```

Comment:
`fakeBuildThread` is a fake for the `buildThread()` function. What you are trying to test here is just that the thread filter function passed to `fakeBuildThread` behaves the way that you expect.

To do that you just need something like:

```js
var annot = {target: [{ selector: {} }]};
assert.equal(threadFilterFn(annot), true);
```